Gemfile.lock

```diff
@@ -7,34 +7,34 @@ GIT
 
 GIT
   remote: https://github.com/CocoaPods/Core.git
-  revision: 167566c0150b8ef9de585cc7bacc664672dc0a54
+  revision: 6900c1b10c50f6201d33061697415d8c19d3a419
```

Comment:
while merging picked up the SHA from Core master which would be 1.7.0 instead of the one from `1-6-stable`

Reply:
that is because `1-6-stable` also points to a `1-6-stable` branch in `cocoapods-core` but in 1.7.0 we want master.

Reply:
figured it out.